lib/matplotlib/tests/test_simplification.py

```diff
@@ -9,7 +9,6 @@
 import matplotlib.pyplot as plt
 
 from pylab import *
```

Comment:
They are removed in the pytest branch as they throw the test finder for a loop (as we would start running parts of the numpy test suite).